frontend/src/css/cat-theme.css

```diff
@@ -187,3 +187,13 @@ Progess Steps
   min-width: 500px;
   max-width: 1000px;
 }
+
+.button-link {
```

Comment:
I couldn't find this in Auora which means it shouldn't be in this theme file.
If this is an inline style you wouldn't need importants.

Reply:
Or you could use an Auora class `btn-secondary`: https://design.gccollab.ca/component/buttons

Reply:
I was trying to aim for the "link" look in the original look and feel, but I'll do the btn-secondary for now and we can change it if Joey doesn't like the look

Reply:
Changed in 21f3abb